packages/components/src/tooltip/test/index.js

```diff
@@ -197,60 +167,44 @@ describe( 'Tooltip', () => {
 		} );
 
 		it( 'should show tooltip when an element is disabled', async () => {
-			const user = userEvent.setup( {
-				advanceTimers: jest.advanceTimersByTime,
-			} );
+			const user = userEvent.setup();
 
-			const { container } = render(
-				<Tooltip text="Show helpful text here">
-					<button disabled>Click me</button>
+			render(
+				<Tooltip { ...props }>
+					<Button disabled>Click me</Button>
 				</Tooltip>
 			);
 
-			const button = screen.getByRole( 'button', { name: 'Click me' } );
+			const button = screen.getByRole( 'button', { name: /Click me/i } );
+
 			expect( button ).toBeVisible();
 			expect( button ).toBeDisabled();
 
-			// Note: this is testing for implementation details,
-			// but couldn't find a better way.
-			const buttonRect = button.getBoundingClientRect();
-			// eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
-			const eventCatcher = container.querySelector( '.event-catcher' );
-			const eventCatcherRect = eventCatcher.getBoundingClientRect();
-			expect( buttonRect ).toEqual( eventCatcherRect );
-
-			await user.hover( eventCatcher );
```

Comment:
I'm not sure the new version of the test reflects what this was doing. Thoughts?

Reply:
I maybe be misunderstanding, or perhaps the test is mislabelled and an additional test is needed? When I see the text `should show tooltip when an element is disabled`, I assumed that is all that needs to be checked. 

And with the new tooltip, we won't have the 'event catcher' so then the test wouldn't translate to the new one. So maybe it would be best to separate this and then leave the implementation detail for a separate test that won't be used for the new tooltip?

Reply:
To clarify, do you mean the suggestion of separating the test into two: one for checking a disabled element and a new one to test event handling? 

Or the original change?

Reply:
I meant the original change. Feel free to do as you prefer with regards to separating the change. If we're getting rid of this version of the component soon, it might make sense to have 2 separate tests and then we'd just remove the event catcher one with the new version of the component.